.github/workflows/test-cli.yml

```
@@ -5,20 +5,35 @@ permissions:
+
+      # Electron App
+      - run: npm run electron:build
+      - name: Add Engine
+        timeout-minutes: 1
+        run: |
+          ./dist/linux-unpacked/electron-shogi \
+            --add-engine \
+            ./scripts/dummy-usi-engine "Dummy Engine" \
+            10 \
+            H4sIAAAAAAAAE42Sa0+DMBSG/8v53IXLLjpiTCxx0W+axU/GmALd1ggtgWI2F/67KRY4u+FCApyXvs/bHs4eynjDMwYB8DJwnKoUIy7XQvKRyrVQshzFqcgjxYpklDDNgID9AMEelroQcv1gXiXLOASdQkDvciOUjWBsRcILCDzXJZDwFatSDQGslAIC3yytzOKEmQtqYjn0hEyHyF5Pipi5elJ4QgqHSD7e4x3Pcr27R/tslZrAQqTccHETeq2LWFkJh4wvN8LJmd44WjnGh2PomRg6HDNB9vCMPRy2T69uRbjh8RfugxU6fGxqzJ5hti4qjsBN2VLpMZUOUG/MX8+FPJjLpu7/eC4kttyijXgugUxICMyTbe3I2m15UwunR3B6GT7v3K45j8oiddClP6E/j6mR32vSixKCd1g0U0JZ0dx/gMBrtW3u1RY+cDcXB/Nk1rbZ9DibDmR712bXBGiltZKIboWOHrV1h/drAm/L588nVm6QsZMu9RSddOx3x5y485kFvijZrDxAWvHS5Pj/TmNNgEsWpfyRFemuzVixtOT1LzpIE4pLBQAA
+      - run: cat $HOME/.config/electron-shogi/usi_engine.json | jq .
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)